engine/schema/src/com/cloud/upgrade/dao/Upgrade41100to41110.java

```diff
@@ -66,6 +66,76 @@ public boolean supportsRollingUpgrade() {
     @Override
     public void performDataMigration(Connection conn) {
         updateSystemVmTemplates(conn);
+        markUnnecessarySecureConfigsAsUnsecure(conn);
+    }
+
+    private void markUnnecessarySecureConfigsAsUnsecure(Connection conn) {
+        /*
+         * the following config items where added as 'Secure' in the past. For some this made sense but for the ones below,
```

Comment:
ping @rafaelweingartner do yousee this?

Reply:
Now yes. The formatting is a little odd though. I am not sure if that was done on purpose.